cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -126,11 +127,22 @@ func handle() {
 	}
 
 	infraClusterLabelsMap := parseLabels()
+	var allowedStorageClassesArr []string
+	var allowAllStorageClasses bool
```

Comment:
do we need to variables here? doesn't `len(allowStorageClassesArr) == 0` imply allowAllStorageClasses?

Reply:
Because having an empty string/array represent everything when assigned as a value to a variable that represents a whitelist is confusing and counter intuitive.